app/src/main/java/ch/epfl/polychef/users/User.java

```diff
@@ -70,6 +75,14 @@ public String getUsername() {
         return new ArrayList<>(subscriptions);
     }
 
+    public int getProfilePictureId() {
+        return profilePictureId;
+    }
+
+    public void setProfilePictureId(int profilePictureId) {
+        this.profilePictureId = profilePictureId;
+    }
```

Comment:
Should we check that the index isn't out of bounds?

Reply:
done